cli/command/image/build.go

```diff
@@ -208,8 +210,12 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 			includes = append(includes, ".dockerignore", relDockerfile)
 		}
 
+		compression := archive.Uncompressed
+		if options.compress {
+			compression = archive.Gzip
```

Comment:
Yes, I thought that would be the case, but thanks for confirming (better safe then sorry when making these choices :smile:)